python/cudf/cudf/core/series.py

```diff
@@ -5,6 +5,7 @@
 import functools
 import inspect
 import pickle
+from turtle import begin_fill
```

Comment:
Unused import?

Reply:
The library import isn't done by me. Should I still remove it?

Reply:
Yes please - thanks!